modules/sdk-coin-polygon/src/polygon.ts

```diff
@@ -171,125 +24,6 @@ export class Polygon extends Eth {
     return new TransactionBuilder(coins.get(this.getBaseChain()));
   }
 
-  /**
-   * Get transfer operation for coin
-   * @param recipient recipient info
-   * @param expireTime expiry time
-   * @param contractSequenceId sequence id
-   * @returns {Array} operation array
-   */
-  getOperation(recipient: Recipient, expireTime: number, contractSequenceId: number): (string | Buffer)[][] {
-    return [
-      ['string', 'address', 'uint256', 'bytes', 'uint256', 'uint256'],
-      [
-        'POLYGON',
-        new optionalDeps.ethUtil.BN(optionalDeps.ethUtil.stripHexPrefix(recipient.address), 16),
-        recipient.amount,
-        Buffer.from(optionalDeps.ethUtil.stripHexPrefix(optionalDeps.ethUtil.padToEven(recipient.data || '')), 'hex'),
-        expireTime,
-        contractSequenceId,
-      ],
-    ];
-  }
-
-  /**
-   * Build arguments to call the send method on the wallet contract
-   * @param txInfo
-   */
-  getSendMethodArgs(txInfo: GetSendMethodArgsOptions): SendMethodArgs[] {
-    // Method signature is
-    // sendMultiSig(address toAddress, uint256 value, bytes data, uint256 expireTime, uint256 sequenceId, bytes signature)
-    return [
-      {
-        name: 'toAddress',
-        type: 'address',
-        value: txInfo.recipient.address,
-      },
-      {
-        name: 'value',
-        type: 'uint256',
-        value: txInfo.recipient.amount,
-      },
-      {
-        name: 'data',
-        type: 'bytes',
-        value: optionalDeps.ethUtil.toBuffer(optionalDeps.ethUtil.addHexPrefix(txInfo.recipient.data || '')),
-      },
-      {
-        name: 'expireTime',
-        type: 'uint256',
-        value: txInfo.expireTime,
-      },
-      {
-        name: 'sequenceId',
-        type: 'uint256',
-        value: txInfo.contractSequenceId,
-      },
-      {
-        name: 'signature',
-        type: 'bytes',
-        value: optionalDeps.ethUtil.toBuffer(optionalDeps.ethUtil.addHexPrefix(txInfo.signature)),
-      },
-    ];
-  }
-
-  /**
-   * Helper function for signTransaction for the rare case that SDK is doing the second signature
-   * Note: we are expecting this to be called from the offline vault
-   * @param params.txPrebuild
-   * @param params.prv
-   * @returns {{txHex: string}}
-   */
-  async signFinalPolygon(params: SignFinalOptions): Promise<FullySignedTransaction> {
```

Comment:
BREAKING CHANGE: signFinalPolygon method name is updated to signFinalEthLike so that it can be used for other EthLike coins